30412/Bindila Adrian Ioan/Assignment 1/commands.c

```diff
@@ -0,0 +1,81 @@
+//
+// Created by bindi on 3/2/2020.
+//
+#include <stdio.h>
+#include <string.h>
+#include "list.h"
+
+int readCommand(char *buffer) {
+    //We assign a number to each command
+    //AF - 1
+    //AL - 2
+    //DF - 3
+    //DL - 4
+    //DOOM_THE_LIST - 5
+    //DE - 6
+    //PRINT_ALL - 7
+    //PRINT_F - 8
+    //PRINT_L - 9
+    if (strstr(buffer, "AF")) {
+        return 1;
+    }
+    if (strstr(buffer, "AL")) {
```

Comment:
these should be `else if` constructs, it doesn't make sense to keep checking the same string/buffer over and over again, it will never enter more than 1 case.
Also, please use `strcmp` not `strstr`